src/ecdsa/ellipticcurve.py

```diff
@@ -1492,6 +1550,9 @@ def __mul__(self, other):
         if self.__order:
             # order*2 as a protection for Minerva
             other = other % (self.__order * 2)
+        self._maybe_precompute()
+        if self.__precompute:
```

Comment:
Maybe it's a good idea to return something truthful, like `self.__precompute` itself, and combine these into one.

Reply:
hmm, true